pallets/domains/src/tests.rs

```diff
@@ -432,15 +462,78 @@ fn reserve_domains_should_fail_when_tried_to_insert_too_many_domains() {
                 domain_from(b"domain-one".to_vec()),
                 domain_from(b"domain-two".to_vec()),
                 domain_from(b"domain-three".to_vec()),
-            ];
+            ].try_into().expect("qed; domains vector exceeds the limit");
+
+            // This should panic
```

Comment:
This test will panic in the previous statment, when trying to turn into `BoundedVec`.
qed; domains vector exceeds the limit